### DISCUSSION TITLE
I have some conspiracy about new control interface update.

### DISCUSSION THREAD

**Discussion by envyska** [seerge/g-helper#709] [General]
Asus have recently updated new control interface version 3.1.1.14.0

I am sure that this update targets g-helper.

If you update control interface, all tweaks from g-helper wont work properly except cpu undervolting(recent one).

Ive read many 'new issues' here. and realised my g14 had same issue.

I had power limit issue, custom fan curve not working issue.

I couldnt switch eco from standard.

after deleting control interface version 3.1.1.14.0 and reinstalling 3.1.1.13.0

all these problems have solved at once.

If you have some g-helper not working issues. dont bother seerge like me KEKW. reinstall control interface version

![1](https://github.com/seerge/g-helper/assets/136965993/07e0328a-7764-4608-a1e8-a48df8b6a7ca)

try to delete control interface V3 then reinstall 3.1.1.13.0 version. Not a joke

Device manager -> System device tab -> Delete control interface with driver -> reboot -> reinstall older control interface 3.1.1.13.0

**Comment by seerge** [comment]
I have 3.1.14 and everything works fine on my G14. 

Since Control interface is being updated even by windows itself, lots of users already have it installed, and it would be insane amount of complains on reddit / here if it would break things.

But what I can imagine, is that Control interface installation was corrupt (by some reason) on your device, so reinstalling it (even tho older version) has fixed things.

Try to install 3.1.14 now and see if still works. 

----

P.S. Also keep in mind, control interface is just a proxy-layer between BIOS (where all actually commands/settings sit) and app that sets them (i.e. g-helper / AC / whatever). There is Linux for example, where similar AsusCtrl app works absolutely fine w/o any "control interfaces" :))

**Reply by seerge** [reply]
![Screenshot 2023-07-01 200836](https://github.com/seerge/g-helper/assets/5920850/68782ce3-d225-4205-98d3-55c2b982f735)

**Reply by envyska** [reply]
It was great for me before update. bad for me after update. solved after reverting interface. Im tired of those fan curves too much. As a noob, i will keep this old version till system force me to do update :(.  I love my current situation.

Then new interface might need new bios version?

I didnt update BIOS 309,310 too. cuz some redditor revealed new bios sets gpu throtling limit at 70 for eco mode.

**Reply by seerge** [reply]
I don’t think that new bios needs anything:) it should work as is
